lib/content-services/src/lib/search/components/search-chip-autocomplete-input/search-chip-autocomplete-input.component.ts

```diff
@@ -43,25 +56,38 @@ export class SearchChipAutocompleteInputComponent implements OnInit, OnDestroy {
     allowOnlyPredefinedValues = true;
 
     @Output()
-    optionsChanged: EventEmitter<string[]> = new EventEmitter();
+    optionsChanged: EventEmitter<AutocompleteOption[]> = new EventEmitter();
+
+    @Output()
+    inputChanged: EventEmitter<string> = new EventEmitter();
```

Comment:
same as https://github.com/Alfresco/alfresco-ng2-components/pull/8764/files#r1266678619, just for string